src/VideoReceiver/gstqgcvideosinkbin.c

```diff
@@ -316,7 +317,7 @@ _vsb_get_type(void)
             NULL,
             sizeof(GstQgcVideoSinkBin),
             0,
-            (GInstanceInitFunc)_vsb_init,
+            (GInstanceInitFunc)(void *)_vsb_init,
```

Comment:
Can you do the same here as well and remove the void* cast?

Reply:
Sure thing! Done.